src/Cafe/OS/libs/vpad/vpad.cpp

```diff
@@ -905,11 +905,11 @@ void vpadExport_VPADStopMotor(PPCInterpreter_t* hCPU)
 void vpadExport_VPADSetTPCalibrationParam(PPCInterpreter_t* hCPU)
 {
 	ppcDefineParamU32(channel, 0);
-	ppcDefineParamMEMPTR(params, VPADTPCalibrationParam, 1);
 
 	debugBreakpoint();
 	if (channel < VPAD_MAX_CONTROLLERS)
 	{
+		ppcDefineParamMEMPTR(params, VPADTPCalibrationParam, 1);
```

Comment:
Parameter initialization should be at the beginning of the function. There is also no performance gained as this code path is taken 99.99% of time.